Mathlib/Algebra/Order/Antidiag/Nat.lean

```diff
@@ -239,4 +241,69 @@ theorem card_finMulAntidiag_of_squarefree {d n : ℕ} (hn : Squarefree n) :
     ArithmeticFunction.cardDistinctFactors_apply, ← List.card_toFinset, toFinset_factors,
     Finset.card_fin]
 
+@[reducible]
+private def f {n : ℕ} : ∀ a ∈ finMulAntidiag 3 n, ℕ × ℕ := fun a _ => (a 0 * a 1, a 0 * a 2)
+
+private theorem finMulAntidiag_three {n : ℕ} :
```

Comment:
I think this one doesn't need to be private. It could also be moved above the `def f`.